### PR TITLE
fix some bugs

### PR DESCRIPTION
**Pull request by rabbyOnTheMars**

### CODE REVIEW

File: src/ui/views/Dashboard/style.less [hunk 1]
```diff
@@ -833,7 +833,7 @@
 .address-popover {
   width: 344px;
   height: 200px;
-  left: 0px !important;
+  left: -8px !important;
   top: -140px !important;
```
Comment (rabbyOnTheMars) [inline src/ui/views/Dashboard/style.less:837]: before: 
wJWyzr7ArG

after:
rxNMv5559y

File: src/ui/views/Dashboard/style.less [hunk 2]
```diff
@@ -944,7 +944,7 @@
 .switch-popover {
   width: 233px;
   max-height: 308px;
-  left: 28px !important;
+  left: 20px !important;
   .ant-popover-arrow {
```
Comment (rabbyOnTheMars) [inline src/ui/views/Dashboard/style.less:948]: before: 

56nE7TVNfI


after:
gFhlraF8UW

File: src/ui/views/SendToken/index.tsx [hunk 3]
```diff
@@ -303,11 +308,7 @@ const SendToken = () => {
   };
 
   const handleClickBack = () => {
-    if (history.length > 1) {
-      history.goBack();
-    } else {
-      history.replace('/');
-    }
+    history.replace('/');
   };
```
Comment (rabbyOnTheMars) [inline src/ui/views/SendToken/index.tsx:313]: When back from chain management, which replace chain management stack with send token, then there will be two send-back history on stack, therefore just replace to ("/")

File: src/ui/views/ChainManagement/index.tsx [hunk 4]
```diff
@@ -167,9 +167,7 @@ const ChainManagement = () => {
   };
   return (
     <div className="chain-management">
-      <PageHeader onBack={goBack}>
-        {t(connection ? 'All Chain' : 'Chain Management')}
-      </PageHeader>
+      <PageHeader onBack={goBack}>{t('All Chains')}</PageHeader>
       <ChainManagementList />
     </div>
```
Comment (rabbyOnTheMars) [inline src/ui/views/ChainManagement/index.tsx:172]: update to all chains
hRb4Gz3Vf1

File: src/ui/component/ChainSelector/tag.tsx [hunk 5]
```diff
@@ -8,11 +8,16 @@ import './style.less';
...
 }
 
-const ChainSelector = ({ value, onChange }: ChainSelectorProps) => {
-  const [showSelectorModal, setShowSelectorModal] = useState(false);
+const ChainSelector = ({
+  value,
+  onChange,
+  showModal = false,
+}: ChainSelectorProps) => {
+  const [showSelectorModal, setShowSelectorModal] = useState(showModal);
```
Comment (rabbyOnTheMars) [inline src/ui/component/ChainSelector/tag.tsx:21]: Show chains select modal when back from all chain page
qxEXAM6QBG

File: src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx [hunk 6]
```diff
@@ -52,7 +54,7 @@ const CurrentConnection = memo(
       hideModal();
     };
     return (
-      <div className="current-connection">
+      <div className={clsx('current-connection', higherBottom && 'mt-10')}>
         <IconLeftConer
```
Comment (rabbyOnTheMars) [inline src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx:58]: before:
FzYxUe6QD0

after:
1wDQJH8BG7

File: src/ui/component/Popup/index.tsx [hunk 7]
```diff
@@ -21,6 +21,7 @@ const Popup = ({
     closable={closable}
     placement={placement}
     className={clsx('custom-popup', className)}
+    destroyOnClose
```
Comment (rabbyOnTheMars) [inline src/ui/component/Popup/index.tsx:24]: destroy dom after close

File: src/ui/component/ChainSelector/Modal.tsx [hunk 8]
```diff
@@ -86,6 +86,7 @@ const ChainSelectorModal = ({
       drawerStyle={{
         height: maxHeight > 450 ? 450 : maxHeight < 130 ? 130 : maxHeight,
       }}
+      destroyOnClose
```
Comment (rabbyOnTheMars) [inline src/ui/component/ChainSelector/Modal.tsx:89]: destroy dom after close

File: src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx [hunk 9]
```diff
@@ -91,7 +91,10 @@ const CurrentConnection = memo(
                         </div>
                       )}
                   </div>
-                ))}
+                ))
+              ) : (
+                <div className="no-dapp">No recent Dapps</div>
+              )}
```
Comment (rabbyOnTheMars) [inline src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx:97]: xmRk8ilvFA

**Review by rabbyOnTheMars — COMMENTED** [review]
before:
middle_img_v2_16e874ca-1a9f-46c3-b552-eb8436d07e7h

**Review by vvvvvv1vvvvvv — APPROVED** [review]

**Pull request merged**